forc-plugins/forc-doc/src/cli.rs

```diff
@@ -2,10 +2,21 @@
 use clap::Parser;
 use forc_pkg::source::IPFSNode;
 
+const ABOUT: &str = "Forc plugin for building a Sway package's documentation";
+
+forc::cli_examples! {
+    [ Build the docs for a project in the current path => doc ]
+    [ Build the docs for a project in the current path and open it in the browser => doc "--open" ]
+    [ Build the docs for a project located in another path => doc "--manifest-path" "../tests_project2" ]
+    [ Build the docs for the current project exportings private types => doc "--document-private-items" ]
```

Comment:
```suggestion
    [ Build the docs for the current project exporting private types => doc "--document-private-items" ]
```

We probably ought to document what this means somewhere.
I recall having conversations with a previous contributor about `forc doc` and private types so I think I know the context but others may not